src/modules/core/link_timeremap.yml

```diff
@@ -0,0 +1,27 @@
+schema_version: 0.3
+type: link
+identifier: timeremap
+title: Time Remap
+version: 1
+copyright: Meltytech, LLC
+license: LGPLv2.1
+language: en
+tags:
+  - Audio
+  - Video
+description: Remap frames in time.
+parameters:
+  - identifier: map
+    title: Map
+    type: float
+    description: >
+      A map of input frame times to output frame times.
```

Comment:
Should specify the unit of measure for the map time value using the `unit` attribute.